### PR TITLE
Add more details on the release engineers role

### PR DESCRIPTION
**Pull request by NSMyself**

### CODE REVIEW

**Review by daniloaliberti — APPROVED** [review]
👌

File: Cookbook/Technical-Documents/ReleaseProcess.md [hunk 1]
```diff
@@ -12,6 +12,12 @@
     2. Escalating abnormal influx of bugs, so the release, as a whole, can be reassessed.
 4. Making sure they can dedicate enough time for the release. In case this is not possible (due to other squad commitments), please inform the iOS chapter lead.
 
+**The objective is to ship the release build as soon as possible whilst maintaining the quality bar and addressing every bug raised by QA**. When dealing with a particularly complicated bug (one that would require a rather significant effort to address) the release engineer should speak with either Andreaa Papillon (Native Apps' Product Manager) or Lilla Szulyovszky (Native Apps' Delivery Manager) as every release is managed and signed-off by the Native Apps squad. The bug's corresponding feature should disabled altogether using its feature switch, if applicable. Such a bug would subsequently be handled by its respective squad.
```
Comment (MichalTKwiecien) [inline Cookbook/Technical-Documents/ReleaseProcess.md]: `The bug's corresponding feature should disabled altogether using its feature switch`
->
`The bug's corresponding feature should be disabled altogether using its feature switch`

Comment (diegopetrucci) [inline Cookbook/Technical-Documents/ReleaseProcess.md]:
```suggestion
**The objective is to ship the release build as soon as possible whilst maintaining the quality bar and addressing every bug raised by QA**.
When dealing with a particularly complicated bug (one that would require a rather significant effort to address) the release engineer should speak with either Andreaa Papillon (Native Apps' Product Manager) or Lilla Szulyovszky (Native Apps' Delivery Manager) as every release is managed and signed-off by the Native Apps squad.
The bug's corresponding feature should be disabled altogether using its feature switch (if applicable). Such a bug would subsequently be handled by its respective squad.
```

File: Cookbook/Technical-Documents/ReleaseProcess.md [hunk 2]
```diff
@@ -12,6 +12,12 @@
...
+
+Release duties have priority over regular squad duties. Please inform your squad of your unavailability before starting to work in the release. It is a well known fact that twenty percent of your work hours overall are to be allocated towards making the release a reality.
```
Comment (diegopetrucci) [inline Cookbook/Technical-Documents/ReleaseProcess.md]:
```suggestion
Release duties have priority over regular squad duties. Please inform your squad of your unavailability before starting to work in the release. As with every other week, twenty percent of your work hours overall are to be allocated towards making the release a reality.
```

File: Cookbook/Technical-Documents/ReleaseProcess.md [hunk 3]
```diff
@@ -12,6 +12,12 @@
...
+
+There are usually two release engineers working at any given time. It goes without saying that both engineers need to work together and that constant feedback is vital.
```
Comment (diegopetrucci) [inline Cookbook/Technical-Documents/ReleaseProcess.md]:
```suggestion
There are usually two release engineers working at any given time. As both engineers have to work together, please be in constant communication with the other.
```

**Review by diegopetrucci — COMMENTED** [review]
A couple of changes on the general tone of the text, mostly to make it sound slightly less aggressive.

**Review by diegopetrucci — APPROVED** [review]

File: Cookbook/Technical-Documents/ReleaseProcess.md (hunk 3 above)
Comment (NSMyself) [reply Cookbook/Technical-Documents/ReleaseProcess.md]: @diegopetrucci I prefer my version to be honest. We always have to work together, only in this case even more so.

**Comment by NSMyself** [comment]
Anyone mind if i merge this? @diegopetrucci @daniloaliberti @MichalTKwiecien ?

**Review by MichalTKwiecien — APPROVED** [review]

**Pull request merged**